draft/2024-04-24-this-week-in-rust.md

```diff
@@ -151,16 +151,6 @@ Rusty Events between 2024-04-24 - 2024-05-22 🦀
 
 ### Virtual
 
-* 2024-04-17 | Virtual (Cardiff, UK) | [Rust and C++ Cardiff](https://www.meetup.com/rust-and-c-plus-plus-in-cardiff/)
-    * [**Reflections on RustNation UK 2024**](https://www.meetup.com/rust-and-c-plus-plus-in-cardiff/events/300325512/)
-* 2024-04-17 | Virtual (Vancouver, BC, CA) | [Vancouver Rust](https://www.meetup.com/vancouver-rust/)
-    * [**Camigo (Peter Kehl)**](https://www.meetup.com/vancouver-rust/events/298542323/)
-* 2024-04-18 | Virtual (Charlottesville, VA, US) | [Charlottesville Rust Meetup](https://www.meetup.com/charlottesville-rust-meetup/)
-    * [**Crafting Interpreters in Rust Collaboratively**](https://www.meetup.com/charlottesville-rust-meetup/events/298368799/)
-* 2024-04-21 | Virtual (Israel) | [Rust in Israel](https://rust.org.il/)
-    * [**Using AstroNvim for Rust development (in Hebrew)**](https://www.meetup.com/code-mavens/events/300265648/)
-* 2024-04-23 | Trondheim, NO | [Rust Trondheim](https://www.meetup.com/rust-trondheim/)
-    * [**Show and Tell in April**](https://www.meetup.com/rust-trondheim/events/300469130/)
 * 2024-04-24 | Virtual + In Person (Prague, CZ) | [Rust Czech Republic](https://www.meetup.com/rust-czech-republic/)
```

Comment:
Will this be relevant when this TWIR goes out, which I imagine will be after this event happens?

Reply:
Fair question, we usually include the "day of" events categorically because they are in the "range" for that particular issue of TWIR, even though the event will have happened by the time the newsletter goes out. The added benefit, is that this serves as a record for the TWIR archives, which can help folks discover groups that they can follow for future events.